src/app/views/settings/Settings.tsx

```diff
@@ -81,6 +82,14 @@ function Settings(props: ISettingsProps) {
 
     if (authenticated) {
       menuItems.push(
+        {
+          key: 'switch-user-app-mode',
+          text: messages[permissionModeType ? "Use Explorer as sample Teams application" : "Use Explorer as logged-in user"],
```

Comment:
```suggestion
          text: translateMessage(permissionModeType 
              ? "Use Explorer as sample Teams application" 
              : "Use Explorer as logged-in user"),
```
Switched to `translateMessage()` + Linebreaks for cleanliness